x/rollup/keeper/msg_server_test.go

```diff
@@ -129,6 +129,7 @@ func (s *KeeperTestSuite) TestApplyUserDeposit() {
 func (s *KeeperTestSuite) TestInitiateWithdrawal() {
 	sender := sdk.AccAddress("addr").String()
 	l1Target := "0x12345abcde"
+	erc20TokenAddress := "0x0123456789abcdef"
```

Comment:
Should we have a separate test for erc20 withdrawals? Interspersing the two seems looks a bit complicated.

Reply:
Yeah I'll split them out, I kept going back and forth on this but I agree that it would be a lot more readable if they were split even if a lot of the same code is shared